### ISSUE TITLE
Fix static interface implementations in DocfxAnnotationGenerator

### ISSUE THREAD

**Issue by jskeet** [nodatime/nodatime.org#194]
CheckMembers currently fails:

```text
Unstable release is missing members in docfx:
  NodaTime.AnnualDate.System#Numerics#IEqualityOperators{NodaTime#AnnualDate,NodaTime#AnnualDate,System#Boolean}#op_Equality(NodaTime.AnnualDate,NodaTime.AnnualDate)
  NodaTime.AnnualDate.System#Numerics#IEqualityOperators{NodaTime#AnnualDate,NodaTime#AnnualDate,System#Boolean}#op_Inequality(NodaTime.AnnualDate,NodaTime.AnnualDate)
  NodaTime.AnnualDate.System#Numerics#IComparisonOperators{NodaTime#AnnualDate,NodaTime#AnnualDate,System#Boolean}#op_LessThan(NodaTime.AnnualDate,NodaTime.AnnualDate)
  NodaTime.AnnualDate.System#Numerics#IComparisonOperators{NodaTime#AnnualDate,NodaTime#AnnualDate,System#Boolean}#op_LessThanOrEqual(NodaTime.AnnualDate,NodaTime.AnnualDate)
  NodaTime.AnnualDate.System#Numerics#IComparisonOperators{NodaTime#AnnualDate,NodaTime#AnnualDate,System#Boolean}#op_GreaterThan(NodaTime.AnnualDate,NodaTime.AnnualDate)
  NodaTime.AnnualDate.System#Numerics#IComparisonOperators{NodaTime#AnnualDate,NodaTime#AnnualDate,System#Boolean}#op_GreaterThanOrEqual(NodaTime.AnnualDate,NodaTime.AnnualDate)
  NodaTime.DateInterval.System#Numerics#IEqualityOperators{NodaTime#DateInterval,NodaTime#DateInterval,System#Boolean}#op_Equality(NodaTime.DateInterval,NodaTime.DateInterval)
  NodaTime.DateInterval.System#Numerics#IEqualityOperators{NodaTime#DateInterval,NodaTime#DateInterval,System#Boolean}#op_Inequality(NodaTime.DateInterval,NodaTime.DateInterval)
  NodaTime.Duration.System#Numerics#IAdditionOperators{NodaTime#Duration,NodaTime#Duration,NodaTime#Duration}#op_Addition(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IUnaryPlusOperators{NodaTime#Duration,NodaTime#Duration}#op_UnaryPlus(NodaTime.Duration)
  NodaTime.Duration.System#Numerics#ISubtractionOperators{NodaTime#Duration,NodaTime#Duration,NodaTime#Duration}#op_Subtraction(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IEqualityOperators{NodaTime#Duration,NodaTime#Duration,System#Boolean}#op_Equality(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IEqualityOperators{NodaTime#Duration,NodaTime#Duration,System#Boolean}#op_Inequality(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IComparisonOperators{NodaTime#Duration,NodaTime#Duration,System#Boolean}#op_LessThan(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IComparisonOperators{NodaTime#Duration,NodaTime#Duration,System#Boolean}#op_LessThanOrEqual(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IComparisonOperators{NodaTime#Duration,NodaTime#Duration,System#Boolean}#op_GreaterThan(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IComparisonOperators{NodaTime#Duration,NodaTime#Duration,System#Boolean}#op_GreaterThanOrEqual(NodaTime.Duration,NodaTime.Duration)
  NodaTime.Duration.System#Numerics#IUnaryNegationOperators{NodaTime#Duration,NodaTime#Duration}#op_UnaryNegation(NodaTime.Duration)
  NodaTime.Instant.System#Numerics#IAdditionOperators{NodaTime#Instant,NodaTime#Duration,NodaTime#Instant}#op_Addition(NodaTime.Instant,NodaTime.Duration)
  NodaTime.Instant.System#Numerics#ISubtractionOperators{NodaTime#Instant,NodaTime#Instant,NodaTime#Duration}#op_Subtraction(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Instant.System#Numerics#ISubtractionOperators{NodaTime#Instant,NodaTime#Duration,NodaTime#Instant}#op_Subtraction(NodaTime.Instant,NodaTime.Duration)
  NodaTime.Instant.System#Numerics#IEqualityOperators{NodaTime#Instant,NodaTime#Instant,System#Boolean}#op_Equality(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Instant.System#Numerics#IEqualityOperators{NodaTime#Instant,NodaTime#Instant,System#Boolean}#op_Inequality(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Instant.System#Numerics#IComparisonOperators{NodaTime#Instant,NodaTime#Instant,System#Boolean}#op_LessThan(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Instant.System#Numerics#IComparisonOperators{NodaTime#Instant,NodaTime#Instant,System#Boolean}#op_LessThanOrEqual(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Instant.System#Numerics#IComparisonOperators{NodaTime#Instant,NodaTime#Instant,System#Boolean}#op_GreaterThan(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Instant.System#Numerics#IComparisonOperators{NodaTime#Instant,NodaTime#Instant,System#Boolean}#op_GreaterThanOrEqual(NodaTime.Instant,NodaTime.Instant)
  NodaTime.Interval.System#Numerics#IEqualityOperators{NodaTime#Interval,NodaTime#Interval,System#Boolean}#op_Equality(NodaTime.Interval,NodaTime.Interval)
  NodaTime.Interval.System#Numerics#IEqualityOperators{NodaTime#Interval,NodaTime#Interval,System#Boolean}#op_Inequality(NodaTime.Interval,NodaTime.Interval)
  NodaTime.LocalDate.System#Numerics#IAdditionOperators{NodaTime#LocalDate,NodaTime#Period,NodaTime#LocalDate}#op_Addition(NodaTime.LocalDate,NodaTime.Period)
  NodaTime.LocalDate.System#Numerics#IAdditionOperators{NodaTime#LocalDate,NodaTime#LocalTime,NodaTime#LocalDateTime}#op_Addition(NodaTime.LocalDate,NodaTime.LocalTime)
  NodaTime.LocalDate.System#Numerics#ISubtractionOperators{NodaTime#LocalDate,NodaTime#Period,NodaTime#LocalDate}#op_Subtraction(NodaTime.LocalDate,NodaTime.Period)
  NodaTime.LocalDate.System#Numerics#ISubtractionOperators{NodaTime#LocalDate,NodaTime#LocalDate,NodaTime#Period}#op_Subtraction(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDate.System#Numerics#IEqualityOperators{NodaTime#LocalDate,NodaTime#LocalDate,System#Boolean}#op_Equality(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDate.System#Numerics#IEqualityOperators{NodaTime#LocalDate,NodaTime#LocalDate,System#Boolean}#op_Inequality(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDate.System#Numerics#IComparisonOperators{NodaTime#LocalDate,NodaTime#LocalDate,System#Boolean}#op_LessThan(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDate.System#Numerics#IComparisonOperators{NodaTime#LocalDate,NodaTime#LocalDate,System#Boolean}#op_LessThanOrEqual(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDate.System#Numerics#IComparisonOperators{NodaTime#LocalDate,NodaTime#LocalDate,System#Boolean}#op_GreaterThan(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDate.System#Numerics#IComparisonOperators{NodaTime#LocalDate,NodaTime#LocalDate,System#Boolean}#op_GreaterThanOrEqual(NodaTime.LocalDate,NodaTime.LocalDate)
  NodaTime.LocalDateTime.System#Numerics#IEqualityOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,System#Boolean}#op_Equality(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalDateTime.System#Numerics#IEqualityOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,System#Boolean}#op_Inequality(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalDateTime.System#Numerics#IComparisonOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,System#Boolean}#op_LessThan(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalDateTime.System#Numerics#IComparisonOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,System#Boolean}#op_LessThanOrEqual(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalDateTime.System#Numerics#IComparisonOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,System#Boolean}#op_GreaterThan(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalDateTime.System#Numerics#IComparisonOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,System#Boolean}#op_GreaterThanOrEqual(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalDateTime.System#Numerics#IAdditionOperators{NodaTime#LocalDateTime,NodaTime#Period,NodaTime#LocalDateTime}#op_Addition(NodaTime.LocalDateTime,NodaTime.Period)
  NodaTime.LocalDateTime.System#Numerics#ISubtractionOperators{NodaTime#LocalDateTime,NodaTime#Period,NodaTime#LocalDateTime}#op_Subtraction(NodaTime.LocalDateTime,NodaTime.Period)
  NodaTime.LocalDateTime.System#Numerics#ISubtractionOperators{NodaTime#LocalDateTime,NodaTime#LocalDateTime,NodaTime#Period}#op_Subtraction(NodaTime.LocalDateTime,NodaTime.LocalDateTime)
  NodaTime.LocalTime.System#Numerics#IAdditionOperators{NodaTime#LocalTime,NodaTime#Period,NodaTime#LocalTime}#op_Addition(NodaTime.LocalTime,NodaTime.Period)
  NodaTime.LocalTime.System#Numerics#ISubtractionOperators{NodaTime#LocalTime,NodaTime#Period,NodaTime#LocalTime}#op_Subtraction(NodaTime.LocalTime,NodaTime.Period)
  NodaTime.LocalTime.System#Numerics#ISubtractionOperators{NodaTime#LocalTime,NodaTime#LocalTime,NodaTime#Period}#op_Subtraction(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.LocalTime.System#Numerics#IEqualityOperators{NodaTime#LocalTime,NodaTime#LocalTime,System#Boolean}#op_Equality(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.LocalTime.System#Numerics#IEqualityOperators{NodaTime#LocalTime,NodaTime#LocalTime,System#Boolean}#op_Inequality(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.LocalTime.System#Numerics#IComparisonOperators{NodaTime#LocalTime,NodaTime#LocalTime,System#Boolean}#op_LessThan(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.LocalTime.System#Numerics#IComparisonOperators{NodaTime#LocalTime,NodaTime#LocalTime,System#Boolean}#op_LessThanOrEqual(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.LocalTime.System#Numerics#IComparisonOperators{NodaTime#LocalTime,NodaTime#LocalTime,System#Boolean}#op_GreaterThan(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.LocalTime.System#Numerics#IComparisonOperators{NodaTime#LocalTime,NodaTime#LocalTime,System#Boolean}#op_GreaterThanOrEqual(NodaTime.LocalTime,NodaTime.LocalTime)
  NodaTime.Offset.System#Numerics#IMinMaxValue<NodaTime#Offset>#MinValue
  NodaTime.Offset.System#Numerics#IMinMaxValue<NodaTime#Offset>#MaxValue
  NodaTime.Offset.System#Numerics#IUnaryNegationOperators{NodaTime#Offset,NodaTime#Offset}#op_UnaryNegation(NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IUnaryPlusOperators{NodaTime#Offset,NodaTime#Offset}#op_UnaryPlus(NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IAdditionOperators{NodaTime#Offset,NodaTime#Offset,NodaTime#Offset}#op_Addition(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#ISubtractionOperators{NodaTime#Offset,NodaTime#Offset,NodaTime#Offset}#op_Subtraction(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IEqualityOperators{NodaTime#Offset,NodaTime#Offset,System#Boolean}#op_Equality(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IEqualityOperators{NodaTime#Offset,NodaTime#Offset,System#Boolean}#op_Inequality(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IComparisonOperators{NodaTime#Offset,NodaTime#Offset,System#Boolean}#op_LessThan(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IComparisonOperators{NodaTime#Offset,NodaTime#Offset,System#Boolean}#op_LessThanOrEqual(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IComparisonOperators{NodaTime#Offset,NodaTime#Offset,System#Boolean}#op_GreaterThan(NodaTime.Offset,NodaTime.Offset)
  NodaTime.Offset.System#Numerics#IComparisonOperators{NodaTime#Offset,NodaTime#Offset,System#Boolean}#op_GreaterThanOrEqual(NodaTime.Offset,NodaTime.Offset)
  NodaTime.OffsetDate.System#Numerics#IEqualityOperators{NodaTime#OffsetDate,NodaTime#OffsetDate,System#Boolean}#op_Equality(NodaTime.OffsetDate,NodaTime.OffsetDate)
  NodaTime.OffsetDate.System#Numerics#IEqualityOperators{NodaTime#OffsetDate,NodaTime#OffsetDate,System#Boolean}#op_Inequality(NodaTime.OffsetDate,NodaTime.OffsetDate)
  NodaTime.OffsetDateTime.System#Numerics#IAdditionOperators{NodaTime#OffsetDateTime,NodaTime#Duration,NodaTime#OffsetDateTime}#op_Addition(NodaTime.OffsetDateTime,NodaTime.Duration)
  NodaTime.OffsetDateTime.System#Numerics#ISubtractionOperators{NodaTime#OffsetDateTime,NodaTime#Duration,NodaTime#OffsetDateTime}#op_Subtraction(NodaTime.OffsetDateTime,NodaTime.Duration)
  NodaTime.OffsetDateTime.System#Numerics#IEqualityOperators{NodaTime#OffsetDateTime,NodaTime#OffsetDateTime,System#Boolean}#op_Equality(NodaTime.OffsetDateTime,NodaTime.OffsetDateTime)
  NodaTime.OffsetDateTime.System#Numerics#IEqualityOperators{NodaTime#OffsetDateTime,NodaTime#OffsetDateTime,System#Boolean}#op_Inequality(NodaTime.OffsetDateTime,NodaTime.OffsetDateTime)
  NodaTime.OffsetTime.System#Numerics#IEqualityOperators{NodaTime#OffsetTime,NodaTime#OffsetTime,System#Boolean}#op_Equality(NodaTime.OffsetTime,NodaTime.OffsetTime)
  NodaTime.OffsetTime.System#Numerics#IEqualityOperators{NodaTime#OffsetTime,NodaTime#OffsetTime,System#Boolean}#op_Inequality(NodaTime.OffsetTime,NodaTime.OffsetTime)
  NodaTime.Period.System#Numerics#IAdditionOperators{NodaTime#Period,NodaTime#Period,NodaTime#Period}#op_Addition(NodaTime.Period,NodaTime.Period)
  NodaTime.Period.System#Numerics#ISubtractionOperators{NodaTime#Period,NodaTime#Period,NodaTime#Period}#op_Subtraction(NodaTime.Period,NodaTime.Period)
  NodaTime.Period.System#Numerics#IUnaryNegationOperators{NodaTime#Period,NodaTime#Period}#op_UnaryNegation(NodaTime.Period)
  NodaTime.Period.System#Numerics#IUnaryPlusOperators{NodaTime#Period,NodaTime#Period}#op_UnaryPlus(NodaTime.Period)
  NodaTime.YearMonth.System#Numerics#IComparisonOperators{NodaTime#YearMonth,NodaTime#YearMonth,System#Boolean}#op_LessThan(NodaTime.YearMonth,NodaTime.YearMonth)
  NodaTime.YearMonth.System#Numerics#IComparisonOperators{NodaTime#YearMonth,NodaTime#YearMonth,System#Boolean}#op_LessThanOrEqual(NodaTime.YearMonth,NodaTime.YearMonth)
  NodaTime.YearMonth.System#Numerics#IComparisonOperators{NodaTime#YearMonth,NodaTime#YearMonth,System#Boolean}#op_GreaterThan(NodaTime.YearMonth,NodaTime.YearMonth)
  NodaTime.YearMonth.System#Numerics#IComparisonOperators{NodaTime#YearMonth,NodaTime#YearMonth,System#Boolean}#op_GreaterThanOrEqual(NodaTime.YearMonth,NodaTime.YearMonth)
  NodaTime.YearMonth.System#Numerics#IEqualityOperators{NodaTime#YearMonth,NodaTime#YearMonth,System#Boolean}#op_Equality(NodaTime.YearMonth,NodaTime.YearMonth)
  NodaTime.YearMonth.System#Numerics#IEqualityOperators{NodaTime#YearMonth,NodaTime#YearMonth,System#Boolean}#op_Inequality(NodaTime.YearMonth,NodaTime.YearMonth)
  NodaTime.ZonedDateTime.System#Numerics#IEqualityOperators{NodaTime#ZonedDateTime,NodaTime#ZonedDateTime,System#Boolean}#op_Equality(NodaTime.ZonedDateTime,NodaTime.ZonedDateTime)
  NodaTime.ZonedDateTime.System#Numerics#IEqualityOperators{NodaTime#ZonedDateTime,NodaTime#ZonedDateTime,System#Boolean}#op_Inequality(NodaTime.ZonedDateTime,NodaTime.ZonedDateTime)
  NodaTime.ZonedDateTime.System#Numerics#IAdditionOperators{NodaTime#ZonedDateTime,NodaTime#Duration,NodaTime#ZonedDateTime}#op_Addition(NodaTime.ZonedDateTime,NodaTime.Duration)
  NodaTime.ZonedDateTime.System#Numerics#ISubtractionOperators{NodaTime#ZonedDateTime,NodaTime#Duration,NodaTime#ZonedDateTime}#op_Subtraction(NodaTime.ZonedDateTime,NodaTime.Duration)
  NodaTime.ZonedDateTime.System#Numerics#ISubtractionOperators{NodaTime#ZonedDateTime,NodaTime#ZonedDateTime,NodaTime#Duration}#op_Subtraction(NodaTime.ZonedDateTime,NodaTime.ZonedDateTime)
  NodaTime.TimeZones.ZoneInterval.System#Numerics#IEqualityOperators{NodaTime#TimeZones#ZoneInterval,NodaTime#TimeZones#ZoneInterval,System#Boolean}#op_Equality(NodaTime.TimeZones.ZoneInterval,NodaTime.TimeZones.ZoneInterval)
  NodaTime.TimeZones.ZoneInterval.System#Numerics#IEqualityOperators{NodaTime#TimeZones#ZoneInterval,NodaTime#TimeZones#ZoneInterval,System#Boolean}#op_Inequality(NodaTime.TimeZones.ZoneInterval,NodaTime.TimeZones.ZoneInterval)
```

I believe these are generated by `return $"{GetUid(method.DeclaringType)}.{interfaceMethodNameUid}{GetParameterNames(method.Parameters)}";` in `ReflectionMember`.

Many (not sure whether it's all) of these are implicit interface implementation, whereas I *think* the intention is that this path is taken by explicit interface implementation.

**Comment by jskeet** [comment]
Filtering out static members there leaves:

```text
Unstable release is missing members in docfx:
  NodaTime.Offset.System#Numerics#IMinMaxValue<NodaTime#Offset>#MinValue
  NodaTime.Offset.System#Numerics#IMinMaxValue<NodaTime#Offset>#MaxValue
```

It looks like the UIDs in docfx are `NodaTime.Offset.System#Numerics#IMinMaxValue{NodaTime#Offset}#MinValue` (and similar for MaxValue). So we need to change the angle brackets. Presumably we already do that somewhere else...

**Comment by jskeet** [comment]
Fixed in #195
